public/js/recipeModal.js

```diff
@@ -81,159 +81,92 @@ var testRatingArr = [
 /////      UPPER CASE ARE TO DO ITEMS     /////
 ///////////////////////////////////////////////
 
-
```

Comment:
@ljmccode  should review these changes since she was the one who wrote this code.

Reply:
This was all me too, just organized the code, and updated references to make the values for the AJAX calls work